build/Jenkinsfile

```diff
@@ -3,7 +3,7 @@
 pipeline {
     agent {
         docker {
-            image 'mattermost/mattermost-build-webapp:v5.2.0'
+            image 'mattermost/mattermost-build-webapp:latest'
```

Comment:
Ya. That makes sense. How should we number them though? MM release numbers don't make sense for master because it will quickly become out of date. Maybe just an incrementing number? Like `:dev-1` `:dev-2` ...

Reply:
When we originally created the `v5.2.0` it was meant to be the "version introduced with the release of v5.2.0", with the assumption that we might introduce a `v5.6.0` as we need now but nothing in between.

I agree that the naming here is suboptimal, given the reasonable expectation to look for versions in between. Maybe dates instead?